src/components/alerts-config/alerts-config.modal.html

```diff
@@ -9,12 +9,36 @@
 
     <ul class="list-group">
 
-      <li class="list-group-item" ng-repeat="alert in alerts" ng-class="{active: alert.active}" ng-click="toggleAlert(alert)">
-        <i class="fa" ng-class="alert.active ? 'fa-toggle-on' : 'fa-toggle-off'" />
-        {{alert.label}}
+      <li class="list-group-item" ng-repeat="alert in alerts" ng-class="{active: alert.active}">
+        <div ng-click="toggleAlert(alert)">
+          <i class="fa" ng-class="alert.active ? 'fa-toggle-on' : 'fa-toggle-off'" />
+          {{alert.label}}
+        </div>
+
+        <ul class="list-group recipients-list" ng-show="showRecipientList(alert)">
+
+          <li class="list-group-item" ng-repeat="member in members" ng-show="member.active" ng-click="toggleRecipient(member)">
```

Comment:
I think `ng-if` should be used instead of `ng-show` here. The difference is `ng-if` will remove the element from the dom, `ng-show` will just apply/remove a class with visibility styles.